pkgs/development/libraries/mustache-tcl/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib, fetchzip, tcl, tcllib }:
+
+tcl.mkTclDerivation {
+  pname = "mustache-tcl";
+  version = "1.1.3.4";
+
+  src = fetchzip {
+    url = "https://github.com/ianka/mustache.tcl/archive/refs/tags/v1.1.3.4.zip";
```

Comment:
```suggestion
    url = "https://github.com/ianka/mustache.tcl/archive/refs/tags/v${version}.zip";
```

Reply:
By the way why are you not using fetchFromGitHub ?

Reply:
No reason, I will change to that if you want.